middlewares/memberValidation.js

```diff
@@ -0,0 +1,40 @@
+const jwt = require("jsonwebtoken");
+const User = require("../models/User");
+const ERROR_MESSAGE = require("../constants");
```

Comment:
```suggestion
const jwt = require("jsonwebtoken");

const User = require("../models/User");
const ERROR_MESSAGE = require("../constants");
```

모듈과 파일 불러오는 부분 구분해주면 좋을 것 같습니다-⭕️